api/leaderboard.go

```diff
@@ -27,8 +27,9 @@ import (
 )
 
 const (
-	notFoundError   = "Could not find data for member"
-	defaultPageSize = 20
+	notFoundError     = "Could not find data for member"
+	defaultPageSize   = 20
+	TenantIDHeaderKey = "wildlife-platform-tenant-id"
```

Comment:
Could you please add this to `docs/leaderboard-enrichment.md`?

Reply:
Done!